### ISSUE TITLE
Easily breaking target atack distance

### ISSUE THREAD

**Issue by cassiothacosta** [dalaranwow/dalaran-wow#5645]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: If you pull any melee enemy to some stiffer cliffs, you are able to unable their melee atacks

**Expected behaviour**: Enemy should walk closer to the target;

**Steps to reproduce the problem**:

1. Pull any melee based enemy
2. Get to a stiffer cliff
3. Enemy unable to atack

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

![wowscrnshot_120818_160405](https://user-images.githubusercontent.com/45318052/49689119-9cdcf480-fb03-11e8-92e8-9bd0a3f8d500.jpg)
